qemu/tests/qemu_guest_agent.py

```diff
@@ -2659,6 +2678,9 @@ def log_check(qga_cmd):
 
         self.gagent.guest_file_read(ret_handle)
         log_check("guest-file-read")
+        if params["os_type"] == "linux":
+            # Back up blacklist just for linux.
+            session.cmd(params["black_list_backup"])
```

Comment:
Could you make a function for the same code?
Or you can add it for the end of test as a clean up.